### ISSUE TITLE
[Bug] when changing the database field “import_id” in the “sys_category” table

### ISSUE THREAD

**Issue by morange** [lochmueller/calendarize#814]
Current Behavior
When updating from TYPO3 v11.5.39 to v12.4.20 with the Extension News version 11.4.2, the following error message is displayed in the “Analyze Database Structure”:

`ALTER TABLEsys_categoryCHANGEimport_idimport_idVARCHAR(100) DEFAULT '' NOT NULL Current value: import_id VARCHAR(100) CHARACTER SET utf8mb4 DEFAULT NULL COLLATEutf8mb4_general_ci

**Expected behavior/output**
In the Database Analyzer, I expected the database to be adjusted as described there.

**Environment**
- TYPO3 version(s): 12.4.20
- news version: 11.4.2
- Is your TYPO3 installation set up with Composer (Composer Mode): Composer Mode
- OS: macOS 14.7
-  ddev1.23.4

**Possible Solution**
Executed via phpMyAdmin:
UPDATE sys_category SET import_id="" WHERE ISNULL(import_id);`

I have already been in contact with the News Extension team about this in this ticket:
https://github.com/georgringer/news/issues/2529

**Comment by lochmueller** [comment]
You are right. There should be an upgrade wizard and perhaps, we should prefix the field with "calendarize" or something more unique.

Regards,
Tim

**Comment by lochmueller** [comment]
Hey @morange 
the current version of EXT:calendarize do not extend sys_category anymore. So the problem is solved...
Regards,
Tim
